systemd/ceph-mds@.service.in

```diff
@@ -8,7 +8,7 @@ Wants=network-online.target local-fs.target time-sync.target remote-fs-pre.targe
 [Service]
 EnvironmentFile=-@SYSTEMD_ENV_FILE@
 ExecReload=/bin/kill -HUP $MAINPID
-ExecStart=/usr/bin/ceph-mds -f --id %i --setuser ceph --setgroup ceph
+ExecStart=@CMAKE_INSTALL_PREFIX@/bin/ceph-mds -f --cluster ceph --id %i --setuser ceph --setgroup ceph
```

Comment:
Can you explain why `--cluster ceph` is added here?  It seems to have been done only for the MDS and the monitor, so it's confusing.

Reply:
yeah, i think we've dropped the `--cluster` option in 5659920c7c128cb8d9552580dbe23dd167a56c31 . seems this change partially reverts it.

Reply:
The `--cluster ceph` have been dropped.